Dockerfile

```diff
@@ -8,10 +8,10 @@ RUN mkdir -p /app
 WORKDIR /app
 
 # install npm packages
-COPY package.json /app
-RUN npm install --production
+COPY package.json package-lock.json /app/
+RUN npm ci --production
 
-FROM node:10.17.0-alpine
+FROM node:10.19.0-alpine
```

Comment:
do we need to stick to 10.x version of node? could we try 12.x? it has lower resource usage

Reply:
Parsoid doesn't work[1] with node 12, sadly
---
[1] https://phabricator.wikimedia.org/T222064